src/app/Http/Controllers/API/PresentationController.php

```diff
@@ -124,6 +124,12 @@ public function destroy(Presentation $presentation)
      */
     public function search(Request $request)
     {
-        //
+        $request->validate([
+            'keywords' => 'required'
+        ]);
+
+        return PresentationResource::collection(
+            Presentation::search($request->get('keywords'))->get()
```

Comment:
`$request->get('keywords')` can be replaced by `$request->keywords`

Reply:
okay